app/Domain/Integrations/IntegrationUrlType.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Integrations;
+
+enum IntegrationUrlType: string
```

Comment:
Nit: A docblock with a link to https://auth0.com/docs/get-started/applications/application-settings#application-uris might be useful for anyone in the future who does not know the meaning of these different types